fossunited/foss_hackathon/doctype/foss_hackathon_localhost/foss_hackathon_localhost.py

```diff
@@ -19,6 +19,7 @@ class FOSSHackathonLocalHost(Document):
         )
 
         city: DF.Link | None
+        is_enabled: DF.Check
```

Comment:
nit - i would prefer a better name. for example, `"localhost.is_enabled"` is good but `"localhost.accepting_attendees"` or `localhost.signup_open` or something along those lines makes it clear that this boolean is specifically related to whether or not the localhost is accepting members/sign ups.

For example, `localhost.is_enabled` could also construed to mean that the entire localhost is disabled for some reason 🤷